backend/stream/adhoc/chens_jsonl_minimizer.py

```diff
@@ -0,0 +1,56 @@
+import argparse
```

Comment:
Yes, as discussed offline, I will move all of my scripts to `scripts` (at root dir) since we don't have a complete pipeline yet, and they are _**adhoc**_ scripts